google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -6323,4 +6384,58 @@ public void testStatelessQueriesWithLocation() throws Exception {
       bigQuery.delete(dataset.getDatasetId(), DatasetDeleteOption.deleteContents());
     }
   }
+
+  @Test
+  public void testUniverseDomainWithInvalidUniverseDomain() {
```

Comment:
Should we include a test case with the correct pairing of universe in the json creds and the builder?